tests/unit_tests/test_wave_03.py

```diff
@@ -129,9 +129,13 @@ def test_swap_items_from_their_empty_returns_false():
 
     nobodys_item = Item()
 
+    
     result = fatimah.swap_items(jolie, item_b, nobodys_item)
-
-    raise Exception("Complete this test according to comments below.")
+    
+    assert len(jolie.inventory) == 0
+    assert len(fatimah.inventory) == 3
```

Comment:
How could we alter these to ensure that test for the correct data structures as well? Right now they would pass with any data structure of length 0 and 3.